### PR TITLE
Add some columns to system.clusters

### PR DESCRIPTION
**Pull request by tavplubix**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added new columns with info about `Replicated` database replicas to `system.clusters`: `database_shard_name`, `database_replica_name`, `is_active`. Added an optional `FROM SHARD` clause to `SYSTEM DROP DATABASE REPLICA` query.

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
The edited test has failed.

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 451f02be3980bef99b351be5cc9e570286f745af with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/48548/451f02be3980bef99b351be5cc9e570286f745af/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: src/Storages/System/StorageSystemClusters.cpp [hunk 1]
```diff
@@ -77,6 +85,12 @@ void StorageSystemClusters::writeCluster(MutableColumns & res_columns, const Nam
             res_columns[i++]->insert(pool_status[replica_index].error_count);
             res_columns[i++]->insert(pool_status[replica_index].slowdown_count);
             res_columns[i++]->insert(pool_status[replica_index].estimated_recovery_time.count());
+            res_columns[i++]->insert(address.database_shard_name);
+            res_columns[i++]->insert(address.database_replica_name);
+            if (is_active.empty())
+                res_columns[i++]->insertDefault();
+            else
+                res_columns[i++]->insert(is_active[replica_idx++]);
```
Comment (alexey-milovidov) [inline src/Storages/System/StorageSystemClusters.cpp:93]: This part is difficult to understand: there is replica_index, and also replica_idx, and for some reason, it is incremented for every shard and replica, but only if there is `is_active`...

**Review by alexey-milovidov — COMMENTED** [review]
Everything else LGTM, merge at your convenience.

**Comment by nikitamikhaylov** [comment]
Sorry, missed this...

**Pull request merged**